### ISSUE TITLE
How to monitor one device with multiple ip adresses

### ISSUE THREAD

**Issue by m4rtini2** [louislam/uptime-kuma#4938]
### ⚠️ Please verify that this question has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/uptime-kuma/security/policy)

### 📝 Describe your problem

Hello, 

i can not make dhcp assign the same ip adress to different mac adresses of one device so to monitor such devices i have created a monitor for every device / interface. But this way i have long list with the same device once active and once not active most of the time. It is very seldom the lan and wlan interface are connected at the same time.
Is there a way to monitor a device with multiple interfaces like lan,wlan,bluetooth with one monitor? 
Something like:  Monitor Type: ping / Friendly Name: mediaplayer / Hostname: lan ip, wlan ip, bluetooth ip / fail if no response from any ip after x tries

I found the following posts, but none of them adresses my issue.

- #4795
- #4031
- #3399 

Is there a solution to this problem or is it planed?

### 🐻 Uptime-Kuma Version

1.23.13

### 💻 Operating System and Arch

debian 12

### 🌐 Browser

firefox

### 🖥️ Deployment Environment

Uptime kuma running in a lxc container on proxmox. Installed with the acording proxmox helper script.

**Comment by kaystrobach** [comment]
Group them, once one in the group is down, the group is considered down.

**Comment by m4rtini2** [comment]
This is not a working solution. If one interface in this device group is down the whole group is shown as not accessible. Even if the device is still connected and  working. So the group should display 'up'. With this configuration i have to open the group to see it is up. Also can't find an option to change that behaviour so i assume uptime kuma can't be used for this scenario.
Are there plans to add this feature?

**Comment by apio-sys** [comment]
Your device could have a single resolvable name regardless of it's current IP address and as such you can consider it UP if you can ping your hostname no?

**Comment by m4rtini2** [comment]
That would work if you could get dhcp entries into dns automaticly but i am using pihole and it does not support this. One ip gets the correct name but the other ips gets 'unknown'. I could manualy enter all the static dhcp records into the dns records but this is not maintainable with many devices.
Uptime kuma is just not the right tool for this scenario yet.
Thank you for your help.

**Comment by apio-sys** [comment]
Indeed, UK is probably not the tool that you are looking for in your specific environment. @CommanderStorm I think you can close this issue here for now.

**Comment by DerJochen** [comment]
I ran into the same problem with my laptop (LAN and WIFI). To solve it I created a group for the laptop with an inverted signal. Then I added both interfaces (also inverted) to the group.
This way, when all interfaces are down, they show green and the group turns it to red. If at least one interface is up, it shows as red. This would normally turn the group red, but inverted it now shows the laptop as "up".
